File: old/Chapter3_challenge/Chapter3/ViewController.m
//
//  ViewController.m
//  Chapter3
//
//  Created by Jinwoo Kim on 4/6/23.
//

#import "ViewController.h"
#import "Renderer.h"

@interface ViewController ()
@property (strong) Renderer *renderer;
@end

@implementation ViewController

- (void)loadView {
    self.view = [MTKView new];
}

- (void)viewDidLoad {
    [super viewDidLoad];
    self.renderer = [[Renderer alloc] initWithMetalView:(MTKView *)self.view];
}

@end

File: Chapter3/Chapter3/ViewController.swift
//
//  ViewController.swift
//  Chapter3
//
//  Created by Jinwoo Kim on 7/17/22.
//

import Cocoa
import MetalKit

class ViewController: NSViewController {
    private var renderer: Renderer?

    override func viewDidLoad() {
        super.viewDidLoad()

        let mtkView: MTKView = view as! MTKView
        renderer = .init(mtkView: mtkView)
    }

    override var representedObject: Any? {
        didSet {
        // Update the view, if already loaded.
        }
    }


}

